core/src/main/scala/org/apache/spark/util/HadoopFSUtils.scala

```diff
@@ -57,14 +57,14 @@ private[spark] object HadoopFSUtils extends Logging {
    * @return for each input path, the set of discovered files for the path
    */
   def parallelListLeafFiles(
-    sc: SparkContext,
-    paths: Seq[Path],
-    hadoopConf: Configuration,
-    filter: PathFilter,
-    ignoreMissingFiles: Boolean,
-    ignoreLocality: Boolean,
-    parallelismThreshold: Int,
-    parallelismMax: Int): Seq[(Path, Seq[FileStatus])] = {
+      sc: SparkContext,
+      paths: Seq[Path],
+      hadoopConf: Configuration,
+      filter: PathFilter,
+      ignoreMissingFiles: Boolean,
+      ignoreLocality: Boolean,
+      parallelismThreshold: Int,
+      parallelismMax: Int): Seq[(Path, Seq[FileStatus])] = {
```

Comment:
Let's don't fix the style arbitrarily. You would fix some styles when you fix other codes around. For example it makes backporting harder.